src/plugins/home/public/application/components/guided_onboarding/use_case_card.tsx

```diff
@@ -113,37 +70,29 @@ export const UseCaseCard = ({ useCase }: UseCaseProps) => {
     return http.basePath.prepend(imagePath);
   };
 
-  const onUseCaseSelection = () => {
-    trackUiMetric(METRIC_TYPE.CLICK, `guided_onboarding__use_case__${useCase}`);
-
-    localStorage.setItem(`guidedOnboarding.${useCase}.tourActive`, JSON.stringify(true));
-    application.navigateToApp(constants[useCase].navigateOptions.appId, {
-      path: constants[useCase].navigateOptions.path,
-    });
-  };
-
-  const title = (
+  const titleElement = (
     <EuiTitle size="xs">
```

Comment:
```suggestion
    <EuiText textAlign="center">
```
We want to center the contents of the card, oddly when applying `center` to the `align` prop on the parent `EuiCard` it centers the progress label as well, I think this is the cleanest way to achieve the centered text while keep the progress left aligned?